### DISCUSSION TITLE
Possibilty of sponsorship or donation

### DISCUSSION THREAD

**Discussion by RayZ0rr** [davatorium/rofi#1843] [Q&A]
I really like this utility and have been using it for years. Now reached a stage in life where I can provide support.

Is there a means to do that?

**Comment by DaveDavenport** [comment]
It is kind of the same answer as last time. (https://github.com/davatorium/rofi/discussions/1602). 

I don't have an option for donations/sponsorship.  It is a hobby project.
I would prefer to get help with documentation/site/etc.
